### DISCUSSION TITLE
Electrosmith Daisy Support

### DISCUSSION THREAD

**Discussion by shanebrown5643** [plugdata-team/plugdata#860] [General]
Hello!

I'd like to learn more about embedding patches on a Daisy Seed using Plugdata. It's certainly easy enough to get a patch going and have it play from the Seed, but I'm interested in controls, cv, gate, etc in and out. When I used pd2dsy, there were send and receive objects that could allow for connection to the I/O of the board. Is there something like the @hv_param object for Plugdata?

Thanks in advance.

**Comment by dromer** [comment]
`@hv_param` is not an object, it's a trick that heavy uses to attach to send/receive objects. Like `send led1 @hv_param`

Plugdata usage of heavy is exactly the same as pd2dsy. The specifics of which are documented by Electro-Smith:
https://github.com/electro-smith/DaisyWiki/wiki/Pd2dsy-JSON

If you want to create a custom seed layout you need to create a custom json file that Plugdata can then read during the compilation step.
